### DISCUSSION TITLE
How do parameters specific to menu work ?

### DISCUSSION THREAD

**Discussion by ghost** [willfaught/paige#92] [Q&A]
The the README, there the following information about parameters specific to menus:

```
[paige]
disabled = false
divider_above = false
divider_below = false
header_above = ""
header_below = ""
```

Adding it to `hugo.toml` and then changing the `divider_above` and `divider_below` to `true` seems to have no effect. 

How can I try this feature ?

**Comment by willfaught** [comment]
Sorry, didn't see this until now for some reason.

How did you add it to your hugo.toml? Those params go in the menu params, not in the site params.

**Reply by ghost** [reply]
Closing - I dont remember what I wanted back then :(

**Reply by willfaught** [reply]
Sorry about the delay. :(

**Reply by ghost** [reply]
Np :)
